core/src/main/resources/jenkins/diagnosis/HsErrPidList/index_fr.properties

```diff
@@ -0,0 +1,9 @@
+blurb=Les rapports d''incident JVM suivants ont été trouvés pour cette instance de Jenkins. \
+  Si vous pensez que c''est un problème interne à Jenkins, merci de <a href="https://www.jenkins.io/redirect/report-an-issue">le reporter</a>. \
+  Jenkins s''appuie sur des heuristiques afin de trouver ces fichiers. Pour une découverte plus fiable envisagez s''il vous plaît d''ajouter \
+  <code>-XX:ErrorFile=/path/to/hs_err_pid%p.log</code> en tant qu''argument de la JVM.
+ago=depuis {0}
+Delete=Supprimer
+Date=Date
+Name=Nom
+Java\ VM\ Crash\ Reports=Rapports de Crash Java JM
```

Comment:
What does JM stand for? The French translation of Java VM (i.e., Virtual Machine) is [Machine virtuelle Java](https://fr.wikipedia.org/wiki/Machine_virtuelle_Java), so I cannot see what the "J" in "JM" would stand for (since Java already appears before the acronym in your translation).

Reply:
Oops! This is a typo ! --'
I meant to type "VM"...

Indeed as you suggested I can tranlate the acronyme 'VM' into french ! I will do it tonight 🙂

Reply:
I do not think we need to introduce an inconsistency by having an acronym in the English version and a full word in the French version. But I do not think we should introduce a typo either.